src/components/gcds-banner/gcds-banner.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'gcds-banner',
+  styleUrl: 'gcds-banner.css',
+  shadow: true,
+})
+export class GcdsBanner {
+  @Element() el: HTMLElement;
+
+  /**
+   * Defines banner role.
+   */
+  @Prop() bannerRole?: 'destructive' | 'primary' | 'secondary' | 'warning' = 'primary';
+
+  /**
+   * Defines the max width of the banner content.
+   */
+  @Prop() maxContentWidth?: 'fluid' | 'lg' | 'md' | 'sm' | 'xs' = 'lg';
+
+  /**
+   * Defines if the banner's position is fixed.
+   */
+  @Prop() positionFixed?: boolean;
+
+  render() {
+    const { bannerRole, maxContentWidth, positionFixed } = this;
+
+    return (
+      <Host>
+        <div
+          class={`banner role-${bannerRole} ${positionFixed ? 'is-fixed' : ''}`}
```

Comment:
While testing the banner in the `gcds-header` component, I noticed when `position-fixed` is set to true, the banner will cover up the content on the page when it first loads. Is there a way to add spacing so the banner doesn't overlap content when on the top of the page?

Reply:
I changed the component to use `position: sticky;` instead which will get rid of the overlap.